workshops/portfolio/README.md

```diff
@@ -11,21 +11,70 @@ Your final design looks like this:
 
 > ![](img/final_1.png)
 
-Here is a link to a [live demo](http://output.jsbin.com/fugoki/2)
+Here is a link to a
+<a href="http://output.jsbin.com/fugoki/2" target="_blank">live demo</a>
```

Comment:
Multiple club leaders report that members will click on the links that take them away from the readme and not be able to get back to the readme. Having this open in a new tab solves this problem.

I am opting to keep this as it is.